egs/swbd/s5c/local/rnnlm/run_rescoring2.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+ngram_order=3
+rnndir=exp/rnnlm_lstm_e
+
+. ./utils/parse_options.sh
+. ./cmd.sh
+. ./path.sh
+
+set -e
+
+LM=fsh_sw1_tg
+weight=0.8
+
+for decode_set in eval2000; do
+  dir=exp/chain/tdnn_lstm_1e_sp
+  decode_dir=${dir}/decode_${decode_set}_$LM
+
+  # Lattice rescoring
+  rnnlm/lmrescore_rnnlm_lat_pruned.sh \
+    --cmd "$decode_cmd -l hostname=b*" \
```

Comment:
Actually, on second thoughts I don't think my idea of making this run_rescoring2.sh script into a utility script has much value, because it contains only one script call.
I think what would be better is to put it as a final stage in local/rnnlm/run_lstm.sh, and make the 'decode_dir' an argument to that script (with a default that works as-is for you).  Most of my other comments still apply (e.g. about the name of the decode-dir).

Now my previous request to rename run_lstm.sh to train_lstm.sh is no longer relevant.

BTW I think it would be a good idea to rename rnnlm/lmrescore_rnnlm_lat_pruned.sh to rnnlm/rescore_lat_pruned.sh (and make similar name changes for other scripts).  'lm' and 'rnnlm' are already implied by the location, so no need to include them in the script name.